build.sbt

```diff
@@ -3,13 +3,39 @@ import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
 
 val appName = "income-tax-expenses"
 
-val silencerVersion = "1.7.3"
+val silencerVersion = "1.7.0"
```

Comment:
Why are we downgrading dependencies?

Reply:
Because silencer 1.7.3 requires scala 2.12.13 which does not play nice with the scoverage. Running the check.sh script will show the error.

Reply:
Ok